### DISCUSSION TITLE
Set Torrent Location options

### DISCUSSION THREAD

**Discussion by m3Lith** [jesec/flood#338] [General]
Under "Set Torrent Location" modal, there are two check-boxes.

- Use as Base Path
- Move data

Regarding first: what does it do?
Regarding second: what is the use case of _not_ moving data, but setting a new location?

**Comment by C84186** [comment]
I agree that "Use as base path" is not obvious, and should be documented more explicitly. 

What it means is that if you have a multi-file torrent, eg, 

```
Tangerine.2015.1080p.BluRay.x265-RARBG/
├── RARBG_DO_NOT_MIRROR.exe
├── RARBG.txt
├── Subs
│   └── 3_English.srt
└── Tangerine.2015.1080p.BluRay.x265-RARBG.mp4
```

& you place this under "Movies" , if you don't select "Use as base path", everything will be placed under `Movies/Tangerine.2015.1080p.BluRay.x265-RARBG/RARBG.txt`. If you do select "Use as base path", everything is placed under "Movies"

Eg. `Moves/Tangerine.2015.1080p.BluRay.x265-RARBG/RARBG.txt`.

Usually you don't want to "Set as base path" since a lot of torrents are folders with commonly-named files inside htem
